### DISCUSSION TITLE
How to create a filter component that is decoupled form the table?

### DISCUSSION THREAD

**Discussion by ptrhck** [TanStack/table#2076] [General]
The [documentaion ](https://codesandbox.io/embed/github/tannerlinsley/react-table/tree/master/examples/filtering) illustrates how to set up great filtering options. In line `247`, all filters are rendered to the table. I was wondering if it is possible to render this filter component outside the table? If so, could someone give an example codesandbox? I think this would be great for the documentation as well!

**Comment by HermanNygaard** [comment]
[Here is a CodeSandbox with two examples](https://codesandbox.io/s/react-table-filter-outside-table-bor4f?file=/src/App.js)
